data/mods/Craft_Gunpowder/cgp_recipes.json

```diff
@@ -24,40 +24,6 @@
   ]
 },{
   "type" : "recipe",
-  "result": "8mm_bootleg",
-  "category": "CC_AMMO",
-  "subcategory": "CSC_AMMO_BULLETS",
-  "skill_used": "fabrication",
-  "skills_required": [ "cooking", 2 ],
-  "difficulty": 7,
-  "time": 45000,
-  "book_learn": [[ "recipe_caseless", 5 ] , ["recipe_bullets" , 9 ] , ["manual_pistol", 10 ] , ["manual_rifle", 10 ]],
-  "qualities":[ {"id":"CHEM","level":2} ],
-  "tools": [
-    [
-      [ "press", 50 ],
-      [ "hotplate", 50 ],
-      [ "toolset", 50 ],
-      [ "fire", -1 ]
-    ],
-    [ [ "press", -1 ] ]
-  ],
-  "components": [
-    [ [ "acid", 1 ] ],
-    [ [ "plastic_chunk", 5 ] ],
-    [ [ "oxy_powder", 100 ] ],
-    [ [ "incendiary", 100 ] ],
-    [
-      [ "lead", 75 ],
-      [ "gold_small", 75 ],
-      [ "silver_small", 75 ],
-      [ "tin", 75 ],
-      [ "bismuth", 75 ],
-      [ "solder_wire", 75 ]
-    ]
-  ]
-},{
-  "type" : "recipe",
```

Comment:
Looks like the only real lump in this one is the recipe mismatch that @Reclusive-reptile mentioned here. The altered numbers and such isn't entirely to my personal taste but that doesn't mean we still can't see how it works out in the wild, as it were.